extensions/micrometer-opentelemetry/deployment/src/test/resources/test-logging.properties

```diff
@@ -0,0 +1,4 @@
+#quarkus.log.category."io.quarkus.micrometer".level=DEBUG
+quarkus.log.category."io.quarkus.bootstrap".level=INFO
+#quarkus.log.category."io.quarkus.arc".level=DEBUG
+quarkus.log.category."io.netty".level=INFO
```

Comment:
Why these?

Reply:
This was inherited from the original tests in Micrometer.